src/libs/TransactionUtils.js

```diff
@@ -285,34 +285,44 @@ function getAllReportTransactions(reportID) {
 }
 
 /**
- * Verifies that the provided waypoints are valid
+ * Filters the waypoints which are valid and returns those
  * @param {Object} waypoints
- * @returns {Boolean}
+ * @param {Boolean} reArrangeIndexes
+ * @returns {Object} validated waypoints
  */
-function validateWaypoints(waypoints) {
+function getValidWaypoints(waypoints, reArrangeIndexes = false) {
     const waypointValues = _.values(waypoints);
-
     // Ensure the number of waypoints is between 2 and 25
     if (waypointValues.length < 2 || waypointValues.length > 25) {
-        return false;
+        return {};
```

Comment:
do we get some kind of feedback when the number of waypoints exceeds 25?

Reply:
Feedback is good but I think no feedback is also fine.
When user adds 26th waypoint, they will notice that Next button is disabled.

Reply:
I think I agree that because the `add stop` button is disabled at this point we're fine.

Reply:
ahhh so we probably won't hit this conditional `waypointValues.length > 25` anyway because the user cannot add more than 25 right?